doc/server_API.md

```diff
@@ -120,8 +108,7 @@ On success:
 {
     "whitelists": [
         {
-            "id": 54,
-            "name": "A",
+            "unique_id": "A",
             "plates": [
```

Comment:
Don't return plates with this call - this is costly. Simply return all whitelists names.